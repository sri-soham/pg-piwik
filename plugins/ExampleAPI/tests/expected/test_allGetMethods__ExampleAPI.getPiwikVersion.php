s:5:"0.6.3";