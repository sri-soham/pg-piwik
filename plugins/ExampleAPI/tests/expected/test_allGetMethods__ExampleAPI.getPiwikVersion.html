<table id="ExampleAPI_getPiwikVersion" border="1">
<thead>
	<tr>
		<th>value</th>
	</tr>
</thead>
<tbody>
	<tr>
		<td>0.6.3</td>
	</tr>
</tbody>
</table>
